### PR TITLE
Another small fix from comments-ui merge to dev

### PR DESCRIPTION
**Pull request by nicknapoli82**
Removed redundant imports. Tested working.

Moved Comments component to only render under the *info* tab.
Tested Comments component working for EventDetails

### CODE REVIEW

File: client/components/Comments/CommentsDumb.js [hunk 1]
```diff
@@ -160,14 +167,17 @@ class Comments extends Component {
...
                   <Comment.Text>{comment.content}</Comment.Text>
-                  <Comment.Actions>
-                    <Comment.Action>
-                      Reply{' '}
-                      {comment.comments.length
-                        ? `(${comment.comments.length})`
-                        : ''}
-                    </Comment.Action>
-                  </Comment.Actions>
+                  { this.props.type === 'post' ?
```
Comment (justin-cook-developer) [inline client/components/Comments/CommentsDumb.js]: Why remove threads for events?

File: client/components/Comments/CommentsDumb.js [hunk 2]
```diff
@@ -64,6 +69,8 @@ class Comments extends Component {
       this.setState({ initialLoad: false });
       this.scrollToNewestComment();
     }
+    else if (prevProps.comments.length !== this.props.comments.length)
```
Comment (justin-cook-developer) [inline client/components/Comments/CommentsDumb.js:79]: i don't think you want this. If I was scrolled up, reading some comments, I wouldn't want to be forced back to the bottom.

**Review by justin-cook-developer — DISMISSED** [review]

File: client/components/Comments/CommentsDumb.js (hunk 1 above)
Comment (nicknapoli82) [reply client/components/Comments/CommentsDumb.js]: We don't have to remove threads for events. 
I actually thought we decided that Event comments were not threaded, and more linear.
I can remove that check on 'post'

File: client/components/Comments/CommentsDumb.js (hunk 2 above)
Comment (nicknapoli82) [reply client/components/Comments/CommentsDumb.js:79]: You need to try it. 
This keeps the position of scroll for comments at the point it should be at, even when new comments are loaded. Without this it automatically scrolls the comments-box to the top when new comments are added.

Comment (justin-cook-developer) [reply client/components/Comments/CommentsDumb.js:79]: Interesting. I assume that's one of the side effects of moving Comments into another scrollable component.

Comment (justin-cook-developer) [reply client/components/Comments/CommentsDumb.js:79]: I will check that out in a bit.

**Review by justin-cook-developer — APPROVED** [review]

**Pull request merged**